gprofiler/profiler_state.py

```diff
@@ -40,3 +44,17 @@ def insert_dso_name(self) -> bool:
     @property
     def profiling_mode(self) -> str:
         return self._profiling_mode
+
+    @property
+    def container_names_client(self) -> Optional[ContainerNamesClient]:
+        return self._container_names_client
+
+    def refresh_container_cache(self) -> None:
+        if self._container_names_client:
+            self._container_names_client.refresh_container_names_cache()
+
+    def get_container_name(self, pid: int) -> str:
+        if self._container_names_client:
+            return self._container_names_client.get_container_name(pid)
+        else:
+            return ""
```

Comment:
Any particular reason you moved these wrappers into this class? We only use it to hold the reference - no need to clone methods into it.

if you did it as a way for code sharing (a wrapper, non-None class for the containers client) then it's okay

Reply:
Haha I see that it was my own suggestion :sweat_smile: 

>  the ProfilerState, or GProfiler, can expose a method get_container_name(pid: int) which gets you the name or "".

I'm trying to avoid from making ProfilerState a god object. But alright let's go with it here, it's useful.